MTA-4798: Support for Listing Languages in 7.3.0 (#137)
migtools/mta-documentation: docs/topics/mta-cli-run-single-app.adoc

[id="mta-cli-run-single-app_{context}"]
= Running the {ProductShortName} {CLINameTitle} against an application

You can run the {ProductFullName} {CLINameTitle} against an application. 

.Procedure

. Open a terminal and navigate to the `<{ProductShortName}_HOME>/` directory.

> . Run the following command to list all the languages in an application. 

Reviewer: ```suggestion
. List all the languages in an application by using the following command: 
```

+
[source,terminal,subs="attributes+"]
----

> $ ./{mta-cli} analyze --input <path_to_input> --list-languages

Reviewer 2: NIT same as above

Author: Done, thanks

----

> The listed langauages can have a supported provider or require a custom rule set for analysis.

Reviewer 2: If the provider is not supported, it would require a provider override, as well as custom rules.

Author: Modified the sentence, thanks.

[id="mta-cli-run-single-app_{context}"]
= Running the {ProductShortName} {CLINameTitle} against an application

You can run the {ProductFullName} {CLINameTitle} against an application. 

:FeatureName: The `--list-languages` flag
include::developer-preview-feature.adoc[]

.Procedure

> . Open a terminal and navigate to the `<{ProductShortName}_HOME>/` directory.

Reviewer 3: Regarding this point - it is not mandatory, user can run this specifying absolute path, both ways of usage will work

Author: Thank you Igor! This point in the procedure is out of scope for this PR. Much appreciate your observation.
